src/components/shared/SubpageContentCard.tsx

```diff
@@ -31,7 +32,7 @@ const SubpageContentCard: React.FC<SubpageContentCardProps> = ({
     return (
         <Card {...cardProps} title={title} className={className}>
             {children}
-            {caption && <div className={footer}>{caption}</div>}
+            {caption && <Meta description={caption} />}
```

Comment:
I was having trouble getting the footer to not overlap the table, so I changed this to not being an absolutely positioned footer